.github/workflows/preview-build.yml

```diff
@@ -200,7 +200,7 @@ jobs:
         with:
           prefix: ${{ env.PATH_PREFIX }}
           strict: ${{ fromJSON(inputs.strict != '' && inputs.strict || 'true') }}
-          metadata-only: ${{ fromJSON(inputs.metadata-only != '' && inputs.metadata-only || 'true') }}
+          metadata-only: ${{ fromJSON(inputs.metadata-only != '' && inputs.metadata-only || 'false') }}
```

Comment:
This matches the default value on the inputs, was confusing to read `true` even though this always defaults to `false`.